### DISCUSSION TITLE
next/image ( lighthouse audit ) `Image elements do not have explicit width and height`

### DISCUSSION THREAD

**Discussion by Babbili** [vercel/next.js#38238] [Help]
I'm using [Next Image Component](https://nextjs.org/docs/api-reference/next/image) and i have set the `width` and `height` properties for the Image component,
```
import Image from 'next/image'
import logo from '../assets/logo.png'

return(
  <Image src={logo} width={80} height={80} />
)
```
In the `lighthouse audit` i'm getting `Image elements do not have explicit width and height` eventhough they are set to 80px
and If i inspect the image in the developer tools and hover over the img `src`, i get `Rendered size: 80 x 80 px`

Is there an explanation or maybe i need to add another property to the Image component ?

**Comment by icyJoseph** [comment]
This is not a problem with the image itself, IIRC, but rather with the placeholder placed by `next/image`, and lighthouse not understanding that it is safe. 

Take a look here, https://github.com/GoogleChrome/lighthouse/issues/11631, I believe a fix from lighthouse is on the way out.

**Reply by runoncedev** [reply]
If I understand right that fix is only for SVG images?

**Reply by icyJoseph** [reply]
No, the thing is that an SVG is rendered in place, as a sibling to your image, which is used to prevent layout shifts. This is a transparent element that it is then covered by your actual image, and it shares the dimensions of your image.

You can see this yourself, by inspecting the group of HTML elements rendered by `next/image`, there's a lot to learn there.

The rather funny thing is, that this is made nearly by the same teams, in pursue of better experiences for users, unfortunately, lighthouse keeps flagging this it seems. Funny in the strange, or unexpected, sense.
